### PR TITLE
code change to upgrade openshift from 4.9 to 4.10

### PR DESCRIPTION
**Pull request by joelapatatechaude**

### CODE REVIEW

**Comment by joelapatatechaude** [comment]
A few scripts-crc file got automatically created, I think when I did a instruqt pull.

File: troubleshooting-basics/03-dealing-with-finalizers/assignment.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ notes:
 tabs:
 - title: OpenShift Console
   type: website
-  url: https://console-openshift-console.crc-dzk9v-master-0.crc.${_SANDBOX_ID}.instruqt.io
+  url: https://console-openshift-console.happy.crc.${_SANDBOX_ID}.instruqt.io
```
Comment (lijcam) [inline troubleshooting-basics/03-dealing-with-finalizers/assignment.md:17]: @joelapatatechaude Can we change `happy` to something like `ocp`? Trying to think of something meaningful with fewer characters.

**Review by lijcam — COMMENTED** [review]
Looks good, how's the stability of new environment. Do we still get systems hanging?

File: troubleshooting-basics/03-dealing-with-finalizers/assignment.md (hunk 1 above)
Comment (joelapatatechaude) [reply troubleshooting-basics/03-dealing-with-finalizers/assignment.md]: Yea makes sense, I switched to ocp

**Comment by joelapatatechaude** [comment]
> Looks good, how's the stability of new environment. Do we still get systems hanging?

Started the track this morning (after switching "happy" to "ocp"), no issues. Will keep an eye on that.

**Pull request merged**